src/shared/util.js

```diff
@@ -158,6 +158,14 @@ export const camelize = cached((str: string): string => {
   return str.replace(camelizeRE, (_, c) => c ? c.toUpperCase() : '')
 })
 
+/**
+ * Kebabize a camelCased string
+ */
+const kebabizeRE = /([a-z])([A-Z])/g
+export const kebabize = cached((str: string): string => {
+  return str.replace(kebabizeRE, (m, p1, p2) => `${p1}-${p2}`).toLowerCase()
+})
+
```

Comment:
Since we're no longer using this, can you delete it please?

Reply:
You got it!